Documentation/packaging.md

```diff
@@ -0,0 +1,65 @@
+# packaging rkt
+
+## Build-time dependencies
+
+#### Basic build-time dependencies
+
+- bash
+- go
+- gofmt
+- file
+- git
+- gpg
+
+#### Additional build-time dependencies for the coreos flavor
+
+wget, gpg, mktemp, md5sum, cpio, gzip, unsquashfs, sort
+
+#### Additional build-time dependencies for the src flavor
+
+intltoolize, libtoolize and all systemd dependencies
```

Comment:
You might add a note that dependencies may differ depending on a version of built systemd.